src/components/Messages/Message/index.js

```diff
@@ -47,9 +47,9 @@ const onClickTranscript = async () => {
 const canPrintTranscript = () => {
 	const { transcript, enableTranscriptMobile } = store.state.config.settings;
 	if (isMobile()) {
-		return transcript && enableTranscriptMobile;
+		return transcript && (!store.state.hidePrint || enableTranscriptMobile);
 	}
-	return transcript;
+	return transcript && !store.state.hidePrint;
```

Comment:
Instead of this how about just returning `false` when hidePrint is true?

```
const { hidePrint } = store.state;

if (hidePrint) {
	return false;
}
```